test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -128,3 +128,18 @@ def count_node_in_state(self, cnsumm='up'):
         :returns: value of parameter being queried
         """
         return int(self.node_summary[cnsumm])
+
+    def count_node_State(self, state='UP  B'):
```

Comment:
@vccardenas can you please make the 'State' with a small case 's' in the function name?

Reply:
Done